assets/promptflow/models/bring-your-own-data-chat-qna/model.yaml

```diff
@@ -1,7 +1,7 @@
 path:
-  container_name: models
-  container_path: samples/bring_your_own_data_chat_qna/v6/model_folder
-  storage_name: promptflowsamples
```

Comment:
Please also change below to files to increase the model version by 1:
https://github.com/Azure/azureml-assets/blob/main/assets/promptflow/models/bring-your-own-data-chat-qna/spec.yaml
https://github.com/Azure/azureml-assets/blob/main/assets/promptflow/models/bring-your-own-data-qna/spec.yaml
![image](https://github.com/Azure/azureml-assets/assets/49388944/eb8394e6-db26-4f17-ac50-c65d529d91ce)
change to 7

Reply:
Done!